index.html

```diff
@@ -2,12 +2,82 @@
 <html>
 <head>
 <link href="style.css" rel="stylesheet" type="text/css">
+<script src="https://kit.fontawesome.com/173db1d6d6.js" crossorigin="anonymous"></script>
 <title>Portafolio</title>
 </head>
 <body>
 
-<h1>This is a Heading</h1>
-<p>This is a paragraph.</p>
+    <section>
+
+        <div class="wrapper">
+            <header class="toolbar">
+                <nav>            
+                    <a href="#">
+                    <li class="item">Edahí Gutiérrez </li> </a>        
+                    <li id="item-right" class="item">&#9776;</li>                                
+                </nav>
+            </header>
+            
+        
+            <h1>Hey There. <br>
+                I’m Edahí</h1>
+        
+            <h2 id="intro">I’am a Software Developer</h2>   
+        
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me. </p>
+        
+            <div class="social-group">
+                <ul>
+                    <li class="social">
+                        <a href="http://github.com/edahigure">
+                            <i class="fa fa-github"></i></a>
+                    </li>
+                    <li class="social">
+                        <a href="https://linkedin.com/in/edahigure">
+                            <i class="fa-brands fa-linkedin-in"></i></a>
+                    </li>
+                    <li class="social">
+                        <a href="https://twitter.com/edahigure">
+                            <i class="fa-brands fa-twitter"></i></a>
+                    </li>
+            
+                    <li class="social">
+                        <a href="#">
+                            <i class="fa-brands fa-angellist"></i></a>
+                    </li>
+                    
+                    <li class="social">        
+                        <a href="#">
+                            <i class="fa-solid fa-m"></i>
+                        </a>
+                
+                    </li>
+        
+                    <li class="image"></li>
+        
+                </ul>
+                
+        
+                
+        
+            </div>
+            
+        
+            
+            
+        
+            
+        </div>
+
+
```

Comment:
[OPTIONAL] Please consider improving your code structure. It is a good practice to don't leave much space between your tags. If a tag is inside another one, you shouldn't lave any space at all. At HTML, the default identation is 4 spaces. This means that you can have something like this:
<_section_>
4spaces <_div_> (notice that I didn't give any space between them, because `div` is inside `section`
4spaces<_/div_>

4spaces<_div2_> (now you can give 1 space between div and div2, if you want)
8spaces<_div3_> (notice that div3 is inside div2, and it should have +4 identation)
8spaces<_/div3_>
4spaces<_/div2_>
<_/section_>

You can check more information about this [here](https://www.freecodecamp.org/news/how-to-indent-in-html-and-why-it-is-important/).

Reply:
removed extra spaces